### PR TITLE
feat: created navbar for project and added routing

### PR DESCRIPTION
**Pull request by 2711-bharath**
- Created a navbar using bootstrap
- Added routing to it
- Result

image

### CODE REVIEW

File: src/app/components/utils/navbar/navbar.component.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.navbar-nav {
+    display: flex;
```
Comment (arjunkarthikeyanakka) [inline src/app/components/utils/navbar/navbar.component.scss]: remove this line and the line below

Comment (viswadha24) [reply src/app/components/utils/navbar/navbar.component.scss]: All good

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,7 +1,19 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { HomeComponent } from './components/views/home/home.component';
```
Comment (khssupriya) [inline src/app/app-routing.module.ts]: Can we not import all in one line?

File: src/app/components/utils/navbar/navbar.component.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.navbar-nav {
+    display: flex;
+    align-items: center;
+
+    .nav-item {
+        padding: 0px 8px;
+    
+        .nav-link {
+            color: #505F98;
```
Comment (khssupriya) [inline src/app/components/utils/navbar/navbar.component.scss]: Make global variables for colors ig? So that we can change later easily.

Comment (khssupriya) [reply src/app/components/utils/navbar/navbar.component.scss]: Make a global variable for font-family and size too

**Review by arjunkarthikeyanakka — APPROVED** [review]

**Review by khssupriya — CHANGES_REQUESTED** [review]
Add a new line at the end of files [https://stackoverflow.com/questions/729692](https://stackoverflow.com/questions/729692)

**Review by khssupriya — APPROVED** [review]

**Pull request merged**